Source/Dafny/Resolver.cs

```diff
@@ -13388,11 +13392,12 @@ private class RBranchExpr : RBranch {
       if (expectExtract) {
         // "y := temp.Extract();"
         var lhs = s.Lhss[0];
-        s.ResolvedStatements.Add(
-          new UpdateStmt(s.Tok, s.Tok,
-            new List<Expression>() { lhsExtract },
-            new List<AssignmentRhs>() { new ExprRhs(VarDotMethod(s.Tok, temp, "Extract")) }
-            ));
+        var assignStatement = new UpdateStmt(s.Tok, s.EndTok,
+          new List<Expression>() { lhsExtract },
+          new List<AssignmentRhs>() { new ExprRhs(VarDotMethod(s.Tok, temp, "Extract")) }
+        );
+
+        s.ResolvedStatements.Add(assignStatement);
```

Comment:
No change here, right?

Reply:
Right. Reverted it so that it's not distracting.